analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -128,4 +131,12 @@ class Analyzer(private val config: AnalyzerConfiguration) {
 
         return OrtResult(repository, run)
     }
+
+    private fun locateRepositoryConfigurationFile(absoluteProjectPath: File) =
+            if (GitRepo().getWorkingTree(absoluteProjectPath).isValid()) {
+                val manifestFile = File(absoluteProjectPath, ".repo/manifest.xml").realFile()
```

Comment:
I believe you could also use `File(GitRepo().getWorkingTree(absoluteProjectPath).getInfo().path)` here (but if you do, you should probably introduce `val gitRepoWorkingTree = GitRepo().getWorkingTree(absoluteProjectPath)` to share with the line above).

Reply:
I'm not sure if this would work on Windows because the path is generated using `invariantSeparatorsPath`.

Reply:
It does work on Windows still as it's wrapped in a `File()` object. But I'm a bit undecided here. Maybe it's better to stick with the more explicit code you have. After all it's maybe too much of an implementation detail that `path` in `VcsInfo` is the path to the manifest for `GitRepo`. On the other hand, it might read slightly nicer.

Anyway, if you already use `resolveSibling` below, you may want to also use `absoluteProjectPath.resolve()` here.

Reply:
This also avoid running some Git commands as part of `getInfo()` which are not required here. Done.